README.md

```diff
@@ -968,6 +967,19 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+* <a name="no-nested-modifiers"></a>
+  Avoid nested modifier `if/unless/while/until` usage. Favor `&&/||` if
+  appropriate.
+<sup>[[link](#no-nested-modifiers)]</sup>
```

Comment:
We definitely already recommend this. Is there a redundant area where we've added this in our version?